src/openzaak/components/catalogi/admin/forms.py

```diff
@@ -38,9 +38,12 @@ def __init__(self, *args, **kwargs):
         self.fields[
             "trefwoorden"
         ].help_text += " Gebruik een komma om waarden van elkaar te onderscheiden."
+        self.fields["trefwoorden"].required = True
```

Comment:
Hrm no, this is not intended behaviour - making fields required will only result in people filling in nonsensical data to get rid of the error message.

I also think the above help text is no longer relevant, will verify!